release-notes.md

```diff
@@ -2,6 +2,21 @@
 
 Kubecost Cloud does not release updates on a scheduled timeline like our self-hosted product. Instead, Kubecost Cloud product updates will be documented here periodically.
 
+## 4/25/24
+
+### New Features:
+* CloudCosts page: performance improvments through pagination and more performant api calls.
+* CloudCosts page: add auto-complete for filters and aggregation by label.

```

Comment:
Refactoring this to match with the corresponding release note for Allocations just a few rows down :) 

```suggestion
* Cloud Costs page: autocomplete now available for filters and aggregations
```